.github/workflows/test.yml

```diff
@@ -15,6 +15,11 @@ on:
 permissions:
   contents: read # to fetch code (actions/checkout)
 
+
+env:
+  FORCE_COLOR: 1
+  NPM_CONFIG_COLOR: always
```

Comment:
this has nothing to do with the required check, but will enable more colorful output in the workflow logs